app/schemas/models/classroom.schema.js

```diff
@@ -116,7 +116,7 @@ _.extend(ClassroomSchema.properties, {
   }, {}),
   stats: c.object({ additionalProperties: true }),
   initializedOuterStats: { type: 'boolean', default: true, description: 'whether the classroom.stats for the classroom is initialized' },
-  type: { title: 'Class Type', type: 'string', enum: ['', 'in-school', 'after-school', 'online', 'camp', 'camp-esports', 'camp-junior', 'homeschool', 'other'] }
+  type: { title: 'Class Type', type: 'string', enum: ['', 'in-school', 'after-school', 'online', 'camp', 'camp-esports', 'camp-junior', 'homeschool', 'other', 'club', 'club-esports', 'club-roblox', 'club-hackstack', 'club-junior'] }
```

Comment:
we only had 4 clubs: AI HS, ozaria, Roblox, esports, why is there a junior in here?  Also, let's not call it club but club-ozaria. Let's not show different options but show an error if we can't create ozaria on coco.

Reply:
![image](https://github.com/user-attachments/assets/07d44c7e-3609-49fa-b9e4-b99a33225ee4)
lol maybe we remove it later?